### ISSUE TITLE
XML parsing for React.js to Solid.js conversion

### ISSUE THREAD

**Issue by rrjanbiah** [Marwes/combine#341]
Combine is recommended in https://www.reddit.com/r/rust/comments/t37twl/hey_rustaceans_got_an_easy_question_ask_here_92022/hyycpd6/?utm_source=reddit&utm_medium=web2x&context=3 

But, I can't seem to find any XML parsing done using Combine. Is that possible and any comments on that?

**Comment by r4v3n6101** [comment]
Of course, it's possible as LL(1) parser can represent XML data by parsing tags. But IMO your choice should be over the `serde` crate as more fitting solution for structs from text conversions and vice versa. Is that appropriate or `combine` is preferable?

**Comment by rrjanbiah** [comment]
@r4v3n6101 Many thanks for your suggestion. Quick glance at `serde` is promising for the React.js to Solid.js conversion. But, I'm worried that it seems to demand hand coding a lot. Anyway, let me closely check that. Thanks again.

**Comment by Marwes** [comment]
Can't have examples for any given format :) . But since xml is a common format there already exists parsers for it so I'd first look at using an existing parser https://crates.io/search?q=xml .

**Comment by rrjanbiah** [comment]
@Marwes Many thanks for your comments and suggestions
